services/tilesets.js

```diff
@@ -16,6 +16,12 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {Object} [query]
```

Comment:
After another look, I think it's better to be consistent with what all the other methods do and that's use config for everything and then pick the attributes for params and query. For example see https://github.com/mapbox/mapbox-sdk-js/blob/master/services/datasets.js#L180-L209.

Reply:
Not sure why I decided to blaze my own trail with that one. :/